package.json

```diff
@@ -116,7 +117,8 @@
     "prettier:fix": "prettier '**/*.js' --write",
     "release": "npmpub",
     "test": "jest --coverage",
-    "watch": "jest --watch"
+    "watch": "jest --watch",
+    "postinstall": "opencollective postinstall"
```

Comment:
FYI: This installs and runs an _opencollective_ CLI script and upon running `npm install` will display a prompt to invite the user to donate after installing your package.